tensorflow_datasets/core/download/checksums.py

```diff
@@ -51,24 +51,43 @@ def __init__(self, size: int, checksum: str):
     self.checksum = checksum
 
   def asdict(self) -> Dict[str, Any]:
-    """Returns the dict representation of the dataclass."""
+    """Returns the dict representation of the dataclass.
```

Comment:
What is the point of writing the same sentence twice ? It seems to me that this detailed description doesn't add any new information to the one-line description. It feels quite redundant.
In this case, it seems best to keep the one-line description

Reply:
Agreed.